schemas/json/layout/layout.schema.v1.json

```
@@ -211,6 +212,17 @@
         }
       }
     },
+    "instantiationButtonComponent": {
+      "properties": {
+        "mapping": {
+          "type": "object",
+          "title": "Mapping",
+          "description": "Mapping from current data model a prefill object being passed to the instantiation api.",
```

I find the sentence somewhat hard to understand. My own attempt:
```suggestion
          "description": "Creates a new app instance with data collected from a stateless part of the app.",
```